src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/NuGet/NuGetRestorer.cs

```diff
@@ -16,7 +16,9 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.NuGet
 {
     using TIdentityDictionary = IImmutableDictionary<NamedIdentity, IComparable>;
 
-    internal class NuGetRestorer : OnceInitializedOnceDisposedAsync
+    [Export(ExportContractNames.Scopes.ConfiguredProject, typeof(IProjectDynamicLoadComponent))]
```

Comment:
This used to be an "unconfigured project service" - it now lives in the configured project scope, which means that one is created for every single configuration. You'll want this to be UnconfiguredProject.